.gitmodules

```diff
@@ -16,10 +16,6 @@
 	path = kubelet
 	url = https://github.com/openshift/kubernetes
 	branch = release-4.19
-[submodule "kube-proxy"]
```

Comment:
you may want to remove it from snyk exclude list. 

https://github.com/openshift/windows-machine-config-operator/blob/39ab1321bfe28f33b3a671f3e91bd8ba9c296103/.snyk#L9

Reply:
same for gofmt 

https://github.com/openshift/windows-machine-config-operator/blob/39ab1321bfe28f33b3a671f3e91bd8ba9c296103/hack/lint-gofmt.sh#L31

Reply:
hacking script

https://github.com/openshift/windows-machine-config-operator/blob/39ab1321bfe28f33b3a671f3e91bd8ba9c296103/docs/HACKING.md?plain=1#L343

Reply:
@jrvaldes WRT to snyk and lint-gofmt:
I was struggling to decide if leaving them there to make it easier to add kube-proxy back to a release branch for a CVE fix. 
I probably now lean towards removing it like your suggestion.

Reply:
Removed